docs/src/simulation_tips.md

```diff
@@ -248,8 +248,38 @@ Error showing value of type OffsetArrays.OffsetArray{Float64,3,CUDA.CuArray{Floa
 ERROR: scalar getindex is disallowed
 ```
 
+Here Julia is telling us that using scalar `getindex` is not `allowed`. So another way to go
+around this limitation is to simply allow scalar operations:
```

Comment:
```suggestion
Here CUDA throws an error because scalar `getindex` is not `allowed`. Another way
around this limitation is to allow scalar operations:
```